pyproject.toml

```diff
@@ -163,6 +163,7 @@ dynamic.dependencies.file = "requirements.txt"
 packages.find = {}
 
 [tool.setuptools_scm]
+local_scheme = "no-local-version"
```

Comment:
Useful for local loading to PyPI

Reply:
i.e. on the same tag

Reply:
just wondering what this setting does? Does it change the version installed locally?